plaidml/bridge/keras/backend_test.py

```diff
@@ -1330,17 +1330,17 @@ def testRandomChanges(self, b):
     # sufficient. It may be worthwhile to make this test more resilient
     # to refactoring and make it test that the update portion is working
     # as expected.
-    @compareForwardClose(skip_tensorflow=True)
+    @compareForwardClose(skip_tensorflow=True, skip_theano=False)
     def testMovingAverageUpdate(self, b):
         return b.moving_average_update(b.variable(m(5, 4, 9, 3, 2)), b.variable(n(5, 4, 9, 3, 2)),
                                        0.95)[1]
 
-    @compareForwardClose(skip_tensorflow=True, atol=1e-6)
+    @compareForwardClose(skip_tensorflow=True, skip_theano=False, atol=1e-6)
     def testBatchNormAndUpdate(self, b):
         b.set_learning_phase(1)
         x = b.variable(n(4, 7))
-        moving_mean = b.variable(m(4, 1))
-        moving_var = b.variable(m(4, 1))
+        moving_mean = b.variable(m(4, 4))
+        moving_var = b.variable(m(4, 4))
```

Comment:
@tzerrell This test was failing once I re-enabled Theano comparisons. The `(4, 1)` shape for these variables weren't matching the resulting shape of `(4, 4)` for the 'target' of the updates (i.e. `moving_mean` and `moving_var`). Does this seem alright to you?